torch/_dynamo/convert_frame.py

```diff
@@ -308,7 +309,8 @@ def transform(instructions, code_options):
             export,
             mutated_closure_cell_contents,
         )
-        tracer.run()
+        with tracing(tracer.output.tracing_context):
+            tracer.run()
```

Comment:
What's going on here?

Reply:
oh you changed the invariant so tracing context is setup while symbolic convert. okey dokey